interface-definitions/service_ids_suricata.xml.in

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="ids">
+        <properties>
+          <help>Intrusion Detection System</help>
+        </properties>
+        <children>
+          <node name="suricata" owner="${vyos_conf_scripts_dir}/service_ids_suricata.py">
+            <properties>
+              <help>Suricata high performance Network IDS, IPS and Network Security Monitoring engine</help>
+              <priority>740</priority>
+            </properties>
+            <children>
+              <leafNode name="interface">
+                <properties>
+                  <help>Interfaces to monitor</help>
+                  <completionHelp>
+                    <script>${vyos_completion_dir}/list_interfaces</script>
+                  </completionHelp>
+                  <multi/>
+                </properties>
+              </leafNode>
```

Comment:
You can include already common XML for [interfaces](https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/generic-interface-multi.xml.i)
```
#include <include/generic-interface-multi.xml.i>
```